pyproject.toml

```diff
@@ -118,3 +119,35 @@ exclude_lines = ["pragma: no cover"]
 [build-system]
 requires = ["setuptools", "poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+exclude = [
+  ".git",
+  ".github",
+  ".hg",
+  ".mypy_cache",
+  ".tox",
+  ".venv",
+  ".env",
+  "venv",
+  "env",
+  "secret",
+  "build",
+  "_build",
+  "buck-out",
+  "dist",
+  "media",
+  "migrations",
+]
+
+# show fixes made in stdout
+show-fixes = false
+
+target-version = "py39"
+
```

Comment:
Should we set a value for `line-length`? Even if we use the same default as Ruff (88), might be good to have it explicitly set.